test/test_torch.py

```diff
@@ -1769,6 +1770,23 @@ def matrixmultiply(mat1, mat2):
             _test_mm(n, m, p, torch.int32, lambda x, y: torch.randint(0, 100, (x, y), dtype=torch.int32))
             _test_mm(n, m, p, torch.int64, lambda x, y: torch.randint(0, 100, (x, y), dtype=torch.int64))
 
+    @unittest.skipIf(
+        not TEST_NUMPY or not TEST_SCIPY, "Numpy or Scipy not found")
+    def test_gelu(self):
+        def _test_gelu(n, m, dtype):
+            def _gelu_ref(X):
+                return X * norm.cdf(X)
+
+            X = np.random.randn(n, m).astype(dtype)
+            res = torch.gelu(torch.Tensor(X))
```

Comment:
once you made the changes above, gelu will be `torch.nn.functional.gelu`

Reply:
Done